src/doc/unstable-book/src/language-features/lang-items.md

```diff
@@ -227,3 +227,99 @@ A third function, `rust_eh_unwind_resume`, is also needed if the `custom_unwind_
 flag is set in the options of the compilation target. It allows customizing the
 process of resuming unwind at the end of the landing pads. The language item's name
 is `eh_unwind_resume`.
+
+## List of all language items
+
+This is a list of all language items in Rust along with where they are located in
+the source code.
+
+- Primitives
+  - `i8`: `libcore/num/mod.rs`
+  - `i16`: `libcore/num/mod.rs`
+  - `i32`: `libcore/num/mod.rs`
+  - `i64`: `libcore/num/mod.rs`
+  - `i128`: `libcore/num/mod.rs`
+  - `isize`: `libcore/num/mod.rs`
+  - `isize`: `libcore/num/mod.rs`
+  - `isize`: `libcore/num/mod.rs`
```

Comment:
looks like `isize` is repeated (same with `usize` below)

Reply:
How did I not see that? (It's defined once for each configuration of pointer size.)